src/Sylius/Bundle/PaymentBundle/Resources/config/routing/integrations/sylius.yaml

```
@@ -0,0 +1,9 @@
+sylius_webhook_payment_method:
+    path: /payment-method/{code}
```

Should it be like so?
```suggestion
    path: /payment-methods/{code}
```